### ISSUE TITLE
Margins in Rmarkdown

### ISSUE THREAD

**Issue by stefanradev93** [alishinski/lavaanPlot#39]
Hi Alex,

I have noticed that using the function `semPlot` in an Rmarkdown produces huge top and bottom white margins, larger than the actual graph. I have been struggling with this for a while. Any ideas on how to fix this?

Thanks!
Stefan

**Comment by alishinski** [comment]
I assume you mean `lavaanPlot`; can you show me an example? What output format are you using with Rmarkdown? If pdf, are you using the `embed_plot_pdf` function?

**Comment by stefanradev93** [comment]
Exactly, I mean `lavaanPlot`. I am using HTML as an output format. You can see an example attached:

![Capture](https://user-images.githubusercontent.com/22372377/164248275-08f8aa1a-9d99-49de-aa21-65f3bff3f545.PNG)

The plot takes up an entire page, compared to all other figures. I am basically doing

`lavaanPlot(model = fit, coefs = T, stand = TRUE)`

**Comment by alishinski** [comment]
So one suggestion I can offer that might help is to use the `save_png` function to save the plot as a png, and then in your .Rmd document you can embed that image. I think that when you save as png it doesn't save it with that big buffer. Let me know whether that helps.

**Comment by rempsyc** [comment]
@stefanradev93 I was able to solve this problem in `rmarkdown` by installing `pdfcrop` in the MiKTeX Console.

![pdfcrop](https://github.com/alishinski/lavaanPlot/assets/13123390/b1773b25-06df-4fc1-b952-a2915a4e94d9)
